device_attributes/index.bs

```diff
@@ -0,0 +1,180 @@
+
+  The <dfn method for="NavigatorManagedData">getSerialNumber()</dfn> method steps are:
+  1. Let |promise| be [=a new promise=].
+  1. Run the following steps in parallel:
+      1. If this API is not called by a managed web application, reject |promise| with a {{NotAllowedError}} DOMException.
+      1. [=Queue a global task=] on the [=relevant global object=] of [=this=] using the [=managed data task source=] to [=resolve=] promise with [[#serial-number]].
+  1. Return |promise|.
+
+# Code example # {#code-example}
+
+Assuming there is a retail enterprise that relies on an online sales system. The backend service pushes different tariffs to the in-store devices based on their annotated location (country, city or sales region) in the morning, and collects sales reports in the afternoon.
+
+With the help of Device Attributes API, this sales application can get the device location by using getAnnotatedLocation() method. The operation will fail if the API call is triggered by a phishing application - looks similar but not the expected one.
+
+<div class="example">
+
+<xmp class="lang-js">
+// Request sensitive data if the current environment is valid.
+function successCallback(location) {
+  const tariff = backend.requestTariff(location);
+  console.log(tariff);
+}
+
+// Stop the workflow if the current environment is unexpected.
+function failureCallback(error) {
+  backend.reportFailure(error);
+  console.error(error.message);
+}
+
+function PrepareTariff() {
+  navigator.managed.getAnnotatedLocation()
+                   .then(successCallback, failureCallback);
+}
+</xmp>
+
+</div>
+
+It is easy to write another similar code snippet to report the sales data including a device serial number by using getAnnotatedAssetId() method. 
+
+NOTE: The website should assume that the data could be tampered with, stolen or reused on another managed device. Alternative security measures should be put in place by the website itself.)
+
+<div class="example">
+
+<xmp class="lang-js">
+function ReportSalesData() {
+  navigator.managed.getAnnotatedAssetId.then(reportCallback);
+}
+</xmp>
+
+</div>
+
+# Security considerations # {#security-considerations}
+
+In accordance with modern security practices, the permission of using these methods is controlled by the administrator per origin. In addition, they are only available to secure contexts on the managed devices.
+
+# Privacy considerations # {#privacy-considerations}
+
+This API provides access to device attributes which will be the same for all origins. (See <a href="https://wicg.github.io/WebApiDevice/managed_config/">MANAGED-CONFIG</a> for per-origin configuration of managed sites.) This could easily be used for tracking purposes if exposed arbitrarily. In a managed environment however, where an administrator is configuring a fleet of machines for particular employees or tasks, it is desirable for these machines to be able to identify themselves to the sites they are expected to be accessing.
```

Comment:
Create a [custom biblio entry](https://tabatkins.github.io/bikeshed/#biblio-custom) so you can create this reference with `[[MANAGED-CONFIG]]`.